Kinvey/Kinvey.xcodeproj/project.pbxproj

```diff
@@ -2764,6 +2764,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = com.kinvey.KinveyAppTests;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_VERSION = 3.0;
```

Comment:
yes, one of our unit test targets was missing the swift version param which causes Xcode keeping ask to migrate every time you open Xcode, so this will stop.